specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Jobs.json

```
@@ -1075,12 +1075,75 @@
             "template": {
               "description": "Job's execution container.",
               "$ref": "#/definitions/JobExecutionTemplate"
+            },
+            "detailedStatus": {
+              "description": "Detailed status of the job execution.",
+              "$ref": "#/definitions/ExecutionStatus"
             }
           },
           "x-ms-client-flatten": true
         }
       }
     },
+    "ExecutionStatus": {
+      "description": "Container Apps Job execution status.",
+      "type": "object",
+      "properties": {
+        "pods": {
+          "description": "Pods in the execution.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PodExecutionStatus"
```

```suggestion
            "$ref": "#/definitions/ReplicaExecutionStatus"
```

to match Vini's suggestion below